Migration.Tool.CLI/MIGRATION_PROTOCOL_REFERENCE.md

```diff
@@ -1,16 +1,15 @@
-## Migration Protocol Reference
+# Migration Protocol Reference
 
 Running the `Migration.Tool.CLI.exe migrate` command (
 see [`Migration.Tool.CLI/README.md`](/Migration.Tool.CLI/README.md)) generates a **migration protocol file**. The
 protocol provides information about the result of the migration, lists required manual steps, etc.
 
-You can find the protocol file in the location specified by
-the `Settings.MigrationProtocolPath` [configuration option](/Migration.Tool.CLI/README.md#Configuration).
+You can find the protocol file path in the `.\Migration.Tool.CLI\appsettings.json` file under the [`MigrationProtocolPath` configuration option](/Migration.Tool.CLI/README.md#Configuration).
```

Comment:
We usually try to write the documentation in a task-oriented style. Although finding the protocol file path is a task we'd like the user to achieve, it is not the primary goal, the goal we'd like them to achieve is to find the protocol file.